### DISCUSSION TITLE
Error on region remove()

### DISCUSSION THREAD

**Discussion by levaweb** [katspaugh/wavesurfer.js#3205] [Q&A] [answered]
I can remove the region, but why am I getting the following error?

```
activeRegion.remove();   // apparently works

TypeError: Cannot read properties of null (reading 'appendChild')

```
Thanks

https://github.com/katspaugh/wavesurfer.js/discussions/3113

**Comment by katspaugh** [comment, marked as answer]
Sounds like you're trying to access a region after deleting it. Can't say much else w/o seeing your code.

**Reply by levaweb** [reply]
Got it, thanks
